internetarchive/openlibrary
# Standardize the Edit button on Books page

Closes #8590 
Feature: This PR aims to standardize the edit button and other admin bar buttons on books page.

### Testing

To verify the changes:
1. Log in as admin.
2. Select any book.
3. Notice the changes to the edit button and admin bar buttons.

### Screenshot

Before:

![before](https://github.com/internetarchive/openlibrary/assets/116485536/28873eed-e620-43a2-9913-460bdd71d3fa)

After:

![after](https://github.com/internetarchive/openlibrary/assets/116485536/2a04dbb4-0b91-4ba6-92e2-7f64180edfad)

### Stakeholders

@mekarpeles @jimchamp
